packages/types/src/schemas/common.ts

```diff
@@ -159,12 +208,20 @@ export const webcontainerSchema = commandsSchema.extend({
       // `false` for disabling the edit link
       z.boolean(),
     ])
-    .optional(),
+    .optional()
+    .describe(
+      'Display a link in lesson for editing the page content. The value is a URL pattern where ${path} is replaced with the lesson’s location relative to src/content/tutorial.',
+    ),
 });
 
 export const baseSchema = webcontainerSchema.extend({
-  title: z.string(),
-  slug: z.string().optional(),
+  title: z.string().describe('The title of the part, chapter, or lesson.'),
+  slug: z
+    .string()
+    .optional()
+    .describe(
+      'Customize the URL segment of this part / chapter or lesson. The full URL path is /:partSlug/:chapterSlug/:lessonSlug.',
```

Comment:
```suggestion
      'Customize the URL segment of this part `/` chapter or lesson. The full URL path is `/:partSlug/:chapterSlug/:lessonSlug`.',
```

Reply:
I think this should be instead:
```suggestion
      'Customize the URL segment of this part, chapter or lesson. The full URL path is `/:partSlug/:chapterSlug/:lessonSlug`.',
```